src/app/container/info-tab/info-tab.component.html

```diff
@@ -82,7 +86,7 @@
         </div>
       </form>
     </li>
-    <li *ngIf="tool?.default_wdl_path || !isPublic">
+    <li *ngIf="(tool?.default_wdl_path || !isPublic) && tool?.mode !== DockstoreToolType.ModeEnum.HOSTED">
```

Comment:
I think this one is fine, maybe just make it non-editable. 
Same with CWL and Dockerfile above

Reply:
So if Hosted, still show the default paths, just hide the edit button?